pkgs/games/gimx/ds4.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/Linux/Dualshock4.xml b/Linux/Dualshock4.xml
```

Comment:
Did you take this patch from upstream? If so please use fetchpatch instead.

Reply:
I didn't.